A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_WHEEL.html

```diff
@@ -253,7 +253,15 @@
             <!-- Markers -->
             <text class="wheel-set-brake-celsius-marker" x="85" y="397">°C</text>
             <text class="wheel-set-brake-rel-marker" x="89" y="420">REL</text>
-
+            <!-- Test temperatures - Uncomment this block if you want to display test values on the lower ECAM -->
+
+            <text id="wheel-testinfo" class="wheel-set-brake-temp" x="570" y="318">Top row of brake temps is for testing purposes</text>
+            <text id="wheel-testinfo" class="wheel-set-brake-temp" x="430" y="358"><--REAL BRAKE TEMPS--></text>
+            <text id="wheel-testinfo" class="wheel-set-brake-temp" x="430" y="398"><--TEMP PROBE TEMPS--></text>
+            <text id="wheel-testbrake-temp-3" class="wheel-set-brake-temp" x="488" y="358">xx</text>
+            <text id="wheel-testbrake-temp-4" class="wheel-set-brake-temp" x="572" y="358">xx</text>
+            <text id="wheel-testbrake-temp-1" class="wheel-set-brake-temp" x="65" y="358">xx</text>
+            <text id="wheel-testbrake-temp-2" class="wheel-set-brake-temp" x="147" y="358">xx</text>
```

Comment:
You should remove this

Reply:
We agreed with tyler that we should keep this for the QA testers, just to make sure everything works correctly regarding the brake cooling simulation. It will be removed before merging